lib/BackgroundJobs/IndexerJob.php

```diff
@@ -72,7 +72,7 @@ public function __construct(
 	) {
 		parent::__construct($time);
 		$this->setInterval($this->getJobInterval());
-		$this->setTimeSensitivity(self::TIME_SENSITIVE);
+		$this->setTimeSensitivity(self::TIME_INSENSITIVE);
```

Comment:
Time insensitive again in order to only run during maintenance window unless run explicitly with `cron.php "OCA\ContextChat\...\IndexerJob"`